klippy/gcode.py

```diff
@@ -27,20 +27,19 @@ def get_commandline(self):
     def get_command_parameters(self):
```

Comment:
thats not part of the config file ... so please remove that change from request

Reply:
wait when did that appear?

Reply:
that doesnt appear on my pr that i put for the official klipper repository just incase https://github.com/Klipper3d/klipper/pull/5287

Reply:
it shows me that you want to change 3 files ... if you go here and show your request and get on "files changed" tab 
probably this is because you not forked this repo and ask to pull and want to pull from master which probably changed sometimes other files thats not needed here ... as far as i understand it should only changed from forks so that really only that files changed which needed